packages/nimble-components/src/rich-text/editor/testing/rich-text-editor.pageobject.ts

```diff
@@ -186,20 +230,27 @@ export class RichTextEditorPageObject {
         await waitForUpdatesAsync();
     }
 
-    public async sliceEditorContent(number: number): Promise<void> {
-        const lastElement = this.getEditorLastChildElement();
-        const text = lastElement.parentElement!.textContent!;
-        lastElement.parentElement!.textContent = text.substring(
-            0,
-            text.length - number
-        );
+    public async sliceEditorContent(from: number, to: number): Promise<void> {
```

Comment:
Does this implementation change address the TODOs in `rich-text-editor-mention.spec.ts` where this method is being called?

Reply:
Yes, this change addresses a couple of TODOs where we used the `sliceEditorContent` methods in the spec file. We are utilizing Tiptap's utilities, which perform the actual deletion of texts in the editor. So I have removed those TODOs from the spec file. In the same way, I have modified for text addition pageobject method too.

Reply:
Added a comment in the issue: https://github.com/ni/nimble/issues/1568#issuecomment-1853282332